specification/gedcom.md

```diff
@@ -393,58 +393,35 @@ For FamilySearch GEDCOM 7.0, this is the `GEDC` structure described in [Chapter
 A header should contain an extension schema structure with tag `SCHMA`
 as described in [Extensions].
 
-##  Extensions
 
-A **standard structure** is a structure whose type, tag, meaning, superstructure, and cardinality within the superstructure are described in this document. This includes records such as `INDI` and substructures such as `INDI`.`NAME`.
+##  Extensions
 
-Two forms of **extension structures** are permitted:
+The structures documented in this specification are called **standard structures**.
+Each standard structure has
 
-- A **tagged extension structure** is a structure whose tag matches production `extTag`. Tagged extension structures may appear as records or substructures of any other structure.
-- An **extended-use standard structure** is a structure whose type, tag, and meaning are defined in this document and whose superstructure is a tagged extension structure.
+- A structure type, identified by a URI
+- A set of locations where it may appear (as a record or as a substructure of a set of superstructure types), with a permitted cardinality in each location
+- A set of substructures it may contain, with a permitted cardinality for each
+- A payload type; if an enumeration, the set of allowed enumerated values
+- A meaning
 
-Extension structures may have substructures, which may be either tagged extension structures of extended-use standard structures.
+Structures must obey all of these rules to conform to this specification.
 
-All other non-standard structures are prohibited. Examples of prohibited structures include, but are not limited to,
+The recommended way to go beyond the set of standard structure types in this specification or to expand their usage is to submit a feature request at on the [FamilySearch GEDCOM development page](https://github.com/FamilySearch/GEDCOM/issues) so that the ramifications of the proposed addition and its interplay with other proposals may be discussed and the addition may be included in a subsequent version of this specification.
```

Comment:
```suggestion
The recommended way to go beyond the set of standard structure types in this specification or to expand their usage is to submit a feature request on the [FamilySearch GEDCOM development page](https://github.com/FamilySearch/GEDCOM/issues) so that the ramifications of the proposed addition and its interplay with other proposals may be discussed and the addition may be included in a subsequent version of this specification.
```
grammar